doc/api/globals.md

```diff
@@ -637,6 +637,51 @@ logical processors available to the current Node.js instance.
 console.log(`This process is running on ${navigator.hardwareConcurrency} logical processors`);
 ```
 
+### `navigator.language`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* {string}
+
+The `navigator.language` read-only property returns a string representing the
+preferred language of the Node.js instance.
+
+The value is representing the language version as defined in [RFC 5646][].
+
+You can override the value of `navigator.language` by setting the
+`LC_ALL` environment variable.
+
+For example:
+
+```sh
+LC_ALL=fr-FR node -p "console.log(navigator.language)"
+```
+
+The fallback value on builds without ICU is `'en-US'`.
+
+```js
+console.log(`The preferred language of the Node.js instance has the tag '${navigator.language}'`);
+```
+
+### `navigator.languages`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* {Array<string>}
+
+The `navigator.languages` read-only property returns an array of strings
+representing the preferred languages of the Node.js instance.
+By default it contains only the value of `navigator.language`.
```

Comment:
So it’s always an array with only one element?

Reply:
Chrome doesn't do that.
If I configure it like this:
![CleanShot 2023-10-30 at 13 57 48](https://github.com/nodejs/node/assets/2352663/a91b6362-3029-45c3-93e3-2f57eb03002d)
I get:
![CleanShot 2023-10-30 at 13 58 19](https://github.com/nodejs/node/assets/2352663/7b018cba-446a-40d2-b0c4-97a93116cccf)

Reply:
Than its done :)